roles/sap_general_preconfigure/tasks/sapnote/assert-2772999.yml

```diff
@@ -4,39 +4,59 @@
   ansible.builtin.debug:
     msg: "SAP note {{ (__sap_general_preconfigure_sapnotes_versions | selectattr('number', 'match', '^2772999$') | first).number }}
           (version {{ (__sap_general_preconfigure_sapnotes_versions | selectattr('number', 'match', '^2772999$') | first).version }}): Configure RHEL 8"
+  tags:
+    - always
 
 - name: Import tasks from '2772999/02-assert-selinux.yml'
   ansible.builtin.import_tasks: 2772999/02-assert-selinux.yml
   when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2772999_02 | d(false)
+  tags:
+    - sap_general_preconfigure_2772999_02
 
 - name: Import tasks from '2772999/03-assert-hostname.yml'
   ansible.builtin.import_tasks: 2772999/03-assert-hostname.yml
   when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2772999_03 | d(false)
+  tags:
+    - sap_general_preconfigure_2772999_03
 
 - name: Import tasks from '2772999/04-assert-network-time-and-date.yml'
   ansible.builtin.import_tasks: 2772999/04-assert-network-time-and-date.yml
   when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2772999_04 | d(false)
+  tags:
+    - sap_general_preconfigure_2772999_04
 
 - name: Import tasks from '2772999/05-assert-firewall.yml'
   ansible.builtin.import_tasks: 2772999/05-assert-firewall.yml
   when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2772999_05 | d(false)
+  tags:
+    - sap_general_preconfigure_2772999_05
 
 - name: Import tasks from '2772999/06-assert-uuidd.yml'
   ansible.builtin.import_tasks: 2772999/06-assert-uuidd.yml
   when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2772999_06 | d(false)
+  tags:
+    - sap_general_preconfigure_2772999_06
 
 - name: Import tasks from '2772999/07-assert-tmpfs.yml'
   ansible.builtin.import_tasks: 2772999/07-assert-tmpfs.yml
   when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2772999_07 | d(false)
+  tags:
+    - sap_general_preconfigure_2772999_07
 
 - name: Import tasks from '2772999/08-assert-linux-kernel-parameters.yml'
   ansible.builtin.import_tasks: 2772999/08-assert-linux-kernel-parameters.yml
   when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2772999_08 | d(false)
+  tags:
+    - sap_general_preconfigure_2772999_08
 
 - name: Import tasks from '2772999/09-assert-process-resource-limits.yml'
   ansible.builtin.import_tasks: 2772999/09-assert-process-resource-limits.yml
   when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2772999_09 | d(false)
+  tags:
+    - sap_general_preconfigure_nproc_limits
```

Comment:
I'm a little bit confused here by the inconsistency. Why not (also) add a tag `sap_general_preconfigure_2772999_09` to this task, similar to the others? And why are there no "theme" tags like `..._kernel_parameters`, etc., here, similar to the other note tasks?

Reply:
This is a coding error which I need to fix.

Reply:
@ja9fuchs BTW The coding error only affects line 56. Regarding the "theme" tags, I am defining those in the files in which the low level files are included, so the tags are in the import tasks of the same name. This should make it easier to understand the code and the related tags.